index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>To-Dos App</title>
+</head>
+  <body>
+      <h1>To-Dos</h1>
+      <form action="#">
+        <input type="text" id="taskInput" placeholder="Enter a task">
+        <input type="submit" value="+">
+      </form>
+      <br>
+      <form id="checkboxForm">
```

Comment:
@jjw18th Why does thid form have an id? Is this id being used anywhere?

Reply:
Not in this html document.